### DISCUSSION TITLE
Unattended mode?

### DISCUSSION THREAD

**Discussion by stephen-bailey-carta** [grahampugh/erase-install#275] [Ideas]
Hey Graham and all! 

I've read a bit about --nointeraction in startosinstall, and was wondering if there could be the same sort of flag. The usecase is that I'm looking to remotely update many zoom room mac minis and absolutely love the erase-install tool, but could do without the "Please click here to continue" window once the image is download, but before it starts the upgrade process. 

I double checked the documentation, and it doesn't look like this is a feature yet. Thanks for your consideration!

**Comment by grahampugh** [comment]
There is no "Please click here to continue" stage unless you use the --confirm option.
